src/watcom/fluid_config.h

```diff
@@ -0,0 +1,36 @@
+/* fluid_config.h will be generated by cmake from this file: src/fluid_config.cmake */
```

Comment:
Ideally, this would be generated by CMake and not committed to the repo, with the version derived from CMakeLists.txt...  If you must leave this file then we should remove this comment.

Reply:
done.